src/lib/types/model.ts

```diff
@@ -176,6 +175,7 @@ export interface IProjectOverview {
     members: number;
     version: number;
     health: number;
+    favorite?: boolean;
```

Comment:
does it have to be optional? can't the store guarantee the false value for old projects?

Reply:
This interface is used in multiple endpoints, not all of them need favorites.